test/e2e/common/runtime.go

```diff
@@ -200,19 +205,29 @@ while true; do sleep 1; done
 				matchTerminationMessage(container, v1.PodFailed, Equal("DONE"))
 			})
 
-			It("should report termination message [LinuxOnly] as empty when pod succeeds and TerminationMessagePolicy FallbackToLogsOnError is set [NodeConformance]", func() {
+			/*
+				Release: v1.14
+				Name: Container Runtime, TerminationMessage, from log output of succeeding pod
+				Description: Create a pod with an container. Container's output is recorded in log and container exits successfully without an error. When container is terminated, terminationMessage MUST have no content as container succeed.
+			*/
+			framework.ConformanceIt("should report termination message [LinuxOnly] as empty when pod succeeds and TerminationMessagePolicy FallbackToLogsOnError is set [NodeConformance]", func() {
 				// Cannot mount files in Windows Containers.
 				container := v1.Container{
 					Image:                    framework.BusyBoxImage,
 					Command:                  []string{"/bin/sh", "-c"},
-					Args:                     []string{"/bin/echo DONE; /bin/true"},
+					Args:                     []string{"/bin/echo -n DONE; /bin/true"},
 					TerminationMessagePath:   "/dev/termination-log",
 					TerminationMessagePolicy: v1.TerminationMessageFallbackToLogsOnError,
 				}
 				matchTerminationMessage(container, v1.PodSucceeded, Equal(""))
 			})
 
-			It("should report termination message [LinuxOnly] from file when pod succeeds and TerminationMessagePolicy FallbackToLogsOnError is set [NodeConformance]", func() {
+			/*
+				Release: v1.14
+				Name: Container Runtime, TerminationMessage, from file
+				Description: Create a pod with an container. Container's output is recorded in a file and the container exits successfully without an error. When container is terminated, terminationMessage MUST match with the content from file.
+			*/
```

Comment:
Thanks for reviewing @timothysc. [LinuxOnly] comment is added into Conformance doc.

Reply:
Regarding whether files cannot be mounted, I think this points to that we need to enable the concept of TerminationMessage on Windows without relying on the specific mechanism (for instance, by mounting a directory in the kubelet on windows and potentially having a different path).   Can you link to a roadmap entry / issue to fix this in the future here?

Reply:
This has been added into KEP.
https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190103-windows-node-support.md

Reply:
> 
> 
> Regarding whether files cannot be mounted, I think this points to that we need to enable the concept of TerminationMessage on Windows without relying on the specific mechanism (for instance, by mounting a directory in the kubelet on windows and potentially having a different path). Can you link to a roadmap entry / issue to fix this in the future here?

Indeed, this will eventually have to be supported on Windows, and we will be able to address this once we have containerd support in. 

Right now, there isn't a feasible way to do it. Indeed, we could instead mount a folder, but then there would be a UX issue: as a User, how do you know where to write the Termination Message? On Linux, it's a bit more straight-forward and obvious, you have a file: ``/dev/termination-file``, you know that that's the right file. But if you mount a folder instead...  ¯\_(ツ)_/¯  You don't see any file there, so there will be some initial confusion about it. Then, after researching and learning what you can do instead, you'll still have to inspect that Pod's configuration to see what the path would be.

Plus, I think there might be some other issues, if multiple volumes would be mounted on the same base path (let's say ``/dev`` would be mounted for the Termination Message file, then another volume would have to be mounted in ``/dev/some-folder``, not sure how this will be handled).